.travis/publishLicenseList.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# NOTE: $GITHUB_TOKEN must be defined as an external system variable (e.g. in the Travis-CI respository settings https://docs.travis-ci.com/user/environment-variables#Defining-Variables-in-Repository-Settings
+#TODO Update user ane email before PR to spdx/license-list-XML
+GITUSERNAME="License Publisher (maintained by Gary O'Neall)"
+GITEMAIL="gary@sourceauditor.com"
+LICENSEDATAREPO="https://github.com/goneall/license-list-data.git"
+HOSTNAMELOC=`expr index "$LICENSEDATAREPO" :// + 2`
+LICENSEDATAURL=https://${GITHUB_TOKEN}@${LICENSEDATAREPO:$HOSTNAMELOC}
+TOOLSJAR=spdx-tools-2.1.8-jar-with-dependencies.jar
+LICENSEXMLDIR=./
+TESTDIR=./test/original
+LICENSEOUTPUTDIR=.OUTPUTFILES
+VERSION=$(git describe)
+if [[ $? != 0 ]]; then VERSION="UNKNOWN"; fi
+RELEASEDATE=$(date -I)
+echo $VERSION
+# Configure git
+git config --global user.email "$GITEMAIL"
+git config --global user.name "$GITUSERNAME"
+# Clone the release
+echo Cloning license list data repository, this could take a while...
+git clone $LICENSEDATAURL $LICENSEOUTPUTDIR --quiet
+# Clean out the old data directories
```

Comment:
We could restructure both scripts into a single make file with different targets.  We could restructure at a later date since it is a rather large change.

Reply:
> We could restructure both scripts into a single make file with different targets. We could restructure at a later date since it is a rather large change.

But I think the `Makefile` should be in license-list-data (since that's what's getting built), not in this repo (which is one of several data sources feeding license-list-data).  See spdx/license-list-data#22.

Simplifying validation doesn't belong in license-list-data though, I think making that part easier calls for a Makefile here.